roles/st2chatops/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- name: Assert st2chatops_hubot_adapter is specified
+  fail:
+    msg: >
+      '"st2chatops_hubot_adapter" must be supplied in roles/st2chatops/defaults/main.yml for st2chatops installation.
```

Comment:
I guess, you mean `https://github.com/StackStorm/ansible-st2/blob/master/roles/st2chatops/vars/main.yml`